src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs

```diff
@@ -37,6 +37,7 @@ static void Main(string[] args)
         Conversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
         Operand: 
           ILocalReferenceOperation: o1 (OperationKind.LocalReference, Type: System.Object) (Syntax: 'o1')
+  IgnoredArguments(0)
```

Comment:
can we not show this if IgnoredArguments is 0 ?

Reply:
We show IgnoredCondition on IDoLoop even when null, so I did the same here.

Reply:
This affects too many tests, which increases the cost of review. It would be good to keep the number of affected tests to a minimum.